data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepper.java

```diff
@@ -15,11 +19,14 @@
  */
 public class DataPrepper {
     private static final Logger LOG = LoggerFactory.getLogger(DataPrepper.class);
+    private static final int SERVER_PORT = 4900;
```

Comment:
Maybe we should make this port also configurable?

Reply:
(Just checked the description)
I agree we need to have a config specific to DataPrepper. This is probably something that should be part of the config file for Data Prepper. We should not use the pipeline.yml file.